docs/docs/cmd/aad/user/user-recyclebinitem-restore.md

```diff
@@ -0,0 +1,99 @@
+# aad user recyclebinitem restore
+
+Removes all users from the tenant recycle bin
+
+## Usage
+
+```sh
+m365 aad user recyclebinitem restore [options]
+```
+
+## Options
+
+`--id <id>`
+: ID of the deleted user.
```

Comment:
Should we specify that this is the ID of the user in the user recycle bin?

Reply:
The ID's of the user, whether they are in the recycle bin or not, won't change, so I'm not sure if this should be needed.

Reply:
Indeed when deleting a user, the user ID doesn't change. The id of the user is the same as the id of the deleted user. I'd opt to leave it as is so we don't make it too complicated by saying that it's the id of the recycle bin item.